### PR TITLE
Feature/mc 9452 create loadbalancer

### PR DESCRIPTION
**Pull request by JamieWulala**

### CODE REVIEW

File: source/includes/gcp/_load_balancer.md [hunk 1]
```diff
@@ -60,6 +60,83 @@ Attributes | &nbsp;
...
+    "urlMap": "https://www.googleapis.com/compute/v1/projects/test-project/global/urlMaps/my-url",
+    "backendService": "https://www.googleapis.com/compute/v1/projects/test-project/global/backendServices/my-backendServices",
+    "backendServices": [
+      "https://www.googleapis.com/compute/v1/projects/cmc-gcp-env-jamie-hvh/global/backendServices/abackendservice"
+    ],
+    "name": "my-loadbalancer",
+    "protocol": "HTTP",
+    "publicAddress": "*.random.net",
+    "reserveStaticIP": false,
+    "shortBackends": "my-backend-service"
```
Comment (SeventhOfNine) [inline source/includes/gcp/_load_balancer.md]: shortBackend

Comment (SeventhOfNine) [inline source/includes/gcp/_load_balancer.md]: this should be a list now, with the changes I made to the fetcher

File: source/includes/gcp/_load_balancer.md [hunk 2]
```diff
@@ -60,6 +60,83 @@ Attributes | &nbsp;
...
+```json
+{
+  "data": {
+    "id": "4546012045076151143",
+     "forwardingRule": "https://www.googleapis.com/compute/v1/projects/test-project/global/forwardingRules/my-forwarding-rule",
+    "targetProxy": "https://www.googleapis.com/compute/v1/projects/test-project/global/targetHttpsProxies/my-target-proxy",
+    "urlMap": "https://www.googleapis.com/compute/v1/projects/test-project/global/urlMaps/my-url",
+    "backendService": "https://www.googleapis.com/compute/v1/projects/test-project/global/backendServices/my-backendServices",
+    "backendServices": [
+      "https://www.googleapis.com/compute/v1/projects/cmc-gcp-env-jamie-hvh/global/backendServices/abackendservice"
```
Comment (y0geshdev) [inline source/includes/gcp/_load_balancer.md]: Missed **cmc-gcp-env-jamie-hvh** , shouldn't it be test-project?

**Review by y0geshdev — APPROVED** [review]
LGTM.

File: source/includes/gcp/_load_balancer.md [hunk 3]
```diff
@@ -60,6 +60,34 @@ Attributes | &nbsp;
...
+
+Create a new load balancer
+
+Required | &nbsp;
+------- | -----------
+`name`<br/>*string* | The display name of the load balancer
+`shortBackend`<br/>*string* | The existing backend serivce that will be attached to this load balancer.
+`shortProtocol`<br/>*string* | The protocol of this resource.
+`shortPort`<br/>*string* | The port number of this resource.
+`shortCertificate`<br/>*string* | The name of the SSL certificate that will be attached to the target proxy if HTTPS is selected. (Requireed if HTTPS is selected for shortProtocol)
```
Comment (simongodard) [inline source/includes/gcp/_load_balancer.md]: typo `Required` with only one `e`

**Review by SeventhOfNine — APPROVED** [review]

**Pull request merged**